zio-http/src/main/scala/zio/http/model/Response.scala

```diff
@@ -12,6 +12,7 @@ import zio.http.service.{
   ClientResponseStreamHandler,
 }
 import zio.http.socket.{SocketApp, WebSocketFrame}
```

Comment:
I think the import of the packages `service` and `socket` are odd.

Reply:
Similar to `Request`.

Maybe `Request` and `Response` should not be moved to the `model` package at all.